src/chainparams.cpp

```diff
@@ -334,10 +291,12 @@ class CRegTestParams : public CChainParams {
         consensus.vDeployments[d].nTimeout = nTimeout;
     }
     void UpdateActivationParametersFromArgs(const ArgsManager& args);
```

Comment:
Since this is called inside of `UpdateFromArgs()` I think this line can be removed?

Reply:
No, the method still needs to be declared.
```
chainparams.cpp:296:6: error: no declaration matches ‘void CRegTestParams::UpdateActivationParametersFromArgs(const ArgsManager&)’
 void CRegTestParams::UpdateActivationParametersFromArgs(const ArgsManager& args)
      ^~~~~~~~~~~~~~
chainparams.cpp:296:6: note: no functions named ‘void CRegTestParams::UpdateActivationParametersFromArgs(const ArgsManager&)’
chainparams.cpp:254:7: note: ‘class CRegTestParams’ defined here
 class CRegTestParams : public CChainParams {
       ^~~~~~~~~~~~~~
chainparams.cpp: In member function ‘void CRegTestParams::UpdateFromArgs(const ArgsManager&)’:
chainparams.cpp:342:5: error: ‘UpdateActivationParametersFromArgs’ was not declared in this scope
     UpdateActivationParametersFromArgs(args);
     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
chainparams.cpp:342:5: note: suggested alternative: ‘UpdateVersionBitsParameters’
     UpdateActivationParametersFromArgs(args);
     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     UpdateVersionBitsParameters

```